packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -779,6 +803,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         private readonly runtimeOptions: Readonly<Required<IContainerRuntimeOptions>>,
         private readonly containerScope: IFluidObject,
         public readonly logger: ITelemetryLogger,
+        public readonly existing: boolean,
```

Comment:
Probably for future (as your PR is not that large):
You might want to consider breaking your PR into multiple, it will be actually faster to get them in :)
First one might be removal of usage of context.existing and ContainerRuntime.existing and changing internals of container to adhere to new realm.

Reply:
That said, it should not be public property, i.e. it should not be a property on this object. As lifetime of this value is only constructor

Reply:
Oh yes, this was just to see what breaks (see description). This is going to be split for sure :D

Reply:
> That said, it should not be public property, i.e. it should not be a property on this object. As lifetime of this value is only constructor

In this change, `instantiateRuntime` needs to check for this flag. But based on your feedback above, that can be replaced with the context's flag so this can be removed completely. Thank you, that makes sense!